### PR TITLE
#331 added __has_include protection to createBriefing and assignGear

### PR DESCRIPTION
**Pull request by darkChozo**
#331 

Tested locally. AssignGear and createBriefing appear to work correctly, and the game doesn't crash if you delete all the affected files. createBriefing looks a bit complex because it needs to bypass the exitWith or it won't detect that a briefing hasn't been loaded.

### CODE REVIEW

File: f/briefing/fn_createBriefing.sqf [hunk 1]
```diff
@@ -34,98 +34,128 @@ if (f_param_debugMode == 1) then
...
 
 // ZEUS
-if (_unitfaction == "" && ! (typeOf player isEqualTo "VirtualSpectator_F")) exitwith {
-	#include "f_briefing_zeus.sqf"
-	["zeus"] call _fnc_debug;
-};
+#if __has_include("f_briefing_zeus.sqf")
+	if (_unitfaction == "" && ! (typeOf player isEqualTo "VirtualSpectator_F")) exitwith {
+		#include "f_briefing_zeus.sqf"
+		[_unitfaction] call _fnc_debug;
```
Comment (shadow-fa) [inline f/briefing/fn_createBriefing.sqf:156]: shouldn't this be "zeus" here?

**Review by shadow-fa — CHANGES_REQUESTED** [review]

File: f/briefing/fn_createBriefing.sqf (hunk 1 above)
Comment (darkChozo) [reply f/briefing/fn_createBriefing.sqf]: good catch

**Review by shadow-fa — APPROVED** [review]
looks good

**Review by NikkoJT — APPROVED** [review]
Looks good to me.

**Pull request merged**